Neos.Utility.ObjectHandling/Classes/TypeHandling.php

```diff
@@ -172,7 +173,7 @@ public static function stripNullableType($type)
     public static function getTypeForValue($value): string
     {
         if (is_object($value)) {
-            if ($value instanceof Proxy) {
+            if ($value instanceof DoctrineProxy || $value insanceof FlowProxy) {
```

Comment:
```suggestion
            if ($value instanceof DoctrineProxy || $value instanceof FlowProxy) {
```

do you even use an IDE? :)
I saw this on my mobile phone and had to start up my computer to complain. But luckily you had added a test that failed anyways

Reply:
Oooops. And proven again: Reading alone is never sufficient - I concentrated on the name and took the key words for granted. :/ Will pay more attention in future.

Reply:
> do you even use an IDE? :)

Haha, rarely for such supposedly simple changes, but maybe I should. But then again that would likely prevent me from actually doing many of those "supposedly simple changes" at all 🤷‍♂️